plugins/trivy/pkg/models/image.go

```diff
@@ -30,6 +30,10 @@ type ImageReport struct {
 	Report         []VulnerabilityList
 }
 
+type TrivyResults struct {
```

Comment:
why do we need this?

Reply:
We update trivy version because old version is not report with all vulns, but trivy report updated their json schema and added a new Results tag.